snake/main.py

```diff
@@ -21,7 +23,7 @@ class Game:
     DEFAULT_RECT = Rect(0, 0, DEFAULT_RECT_SIZE, DEFAULT_RECT_SIZE)
     NUM_UNITS_RESPAWN = 3
     FRAMETIME_WAITED = 64
-
+    score=0
```

Comment:
According to [PEP8](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements), you should keep a single whitespace on both each side of the assignment operator, as I said on your first pull request.